clients/unshaded/src/main/java/tachyon/client/file/TachyonFileSystemUtils.java

```diff
@@ -135,4 +143,43 @@ public static boolean waitCompleted(final TachyonFileSystemCore tfs, final Tachy
 
     return completed;
   }
+
+  /**
+   * Persist the given file to the under file system.
+   *
+   * @param tfs {@link TachyonFileSystem} to carry out tachyon operations
+   * @param file the file to persist
+   * @param fileInfo the file info of the file
+   * @param tachyonConf TachyonConf object
+   * @return the size of the file persisted
```

Comment:
It's not used right now, but if this utility method is to be used later, it could be a nice way of seeing how many bytes were copied. @gpang suggested it.